components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -144,6 +146,21 @@ const SearchView = () => {
             handleClearResults();
         }
 
+        let processedQueryString;
+        if (CLP_STORAGE_ENGINES.CLP_S === Meteor.settings.public.ClpStorageEngine) {
+            processedQueryString = queryString;
+        } else {
+            try {
+                processedQueryString = unquoteString(queryString, '"', '\\');
+                if ("" === processedQueryString) {
+                    throw new Error("Cannot be empty.");
```

Comment:
We're `throw`ing the exception here because supposedly the UI does not permit any submission when the text field is empty, right?

Reply:
The backend doesn't support empty query strings. In the UI, the user can't submit an empty string as their query since the form won't let them. However, the form does let them submit an empty quoted string, so this check is to handle that.

Reply:
Gotcha. 

Would it be better to throw directly in `unquoteString` so that we can avoid `throw`ing in a `try-catch` here?